playbooks/robusta_playbooks/k8s_resource_enrichments.py

```diff
@@ -128,6 +130,7 @@ def to_pod_obj(pod: Pod, cluster: str) -> RelatedPod:
         addresses=addresses,
         containers=get_pod_containers(pod),
         status=pod.status.phase,
+        status_reason=pod.status.reason,
```

Comment:
Can you please change to camel case to be consistent with the other class fields?

Reply:
done